sdk/containerregistry/container-registry/test/utils/utils.ts

```diff
@@ -24,36 +21,29 @@ const replaceableVariables: Record<string, string> = {
 };
 
 const expiryReplacement = "eyJleHAiOjg2NDAwMDAwMDAwMDB9"; //  base64 encoding of '{"exp":8640000000000}' ;
-export const recorderEnvSetup: RecorderEnvironmentSetup = {
+export const recorderStartOptions: RecorderStartOptions = {
   // == Recorder Environment Setup == Add the replaceable variables from
   // above
-  replaceableVariables,
+  envSetupForPlayback,
 
-  // We don't use this in the template, but if we had any query parameters
-  // we wished to discard, we could add them here
-  queryParametersToSkip: [],
-
-  // Finally, we need to remove the AAD `refresh_token` from any requests.
-  // This is very important, as it cannot be removed using environment
-  // variable or query parameter replacement.  The
-  // `customizationsOnRecordings` field allows us to make arbitrary
-  // replacements within recordings.
-  customizationsOnRecordings: [
-    (recording: string): string =>
-      recording.replace(
-        /"refresh_token":"[^"]+"/g,
-        `"refresh_token":"sanitized.${expiryReplacement}.sanitized"`
-      ),
-    (recording: string): string =>
-      recording.replace(/access_token=(.+?)(&|")/, `access_token=access_token$2`),
-    (recording: string): string =>
-      recording.replace(
-        /refresh_token=([^&]+?)(&|")/,
-        `refresh_token=sanitized.${expiryReplacement}.sanitized$2`
-      ),
-  ],
-  onLoadCallbackForPlayback: () => {
-    pluginForClientSecretCredentialTests(env.CONTAINERREGISTRY_TENANT_ID);
+  sanitizerOptions: {
+    generalSanitizers: [
+      {
+        regex: true,
+        target: `"refresh_token":"[^"]+"`,
+        value: `"refresh_token":"sanitized.${expiryReplacement}.sanitized"`,
+      },
+      {
+        regex: true,
+        target: `access_token=([^&]+?)(&|"|$)`,
```

Comment:
I think this was meant for identity traffic - can remove I think.

Reply:
This can't be removed due to ACR's funky auth mechanism (where it exchanges the access token for another token using an ACR API).

Reply:
@jeremymeng can confirm